sql/core/src/test/scala/org/apache/spark/sql/jdbc/JDBCSuite.scala

```diff
@@ -732,15 +732,13 @@ class JDBCSuite extends QueryTest
   test("Pass extra properties via OPTIONS") {
     // We set rowId to false during setup, which means that _ROWID_ column should be absent from
     // all tables. If rowId is true (default), the query below doesn't throw an exception.
-    intercept[JdbcSQLException] {
```

Comment:
why did we fail before?

Reply:
H2 older version will throws:
```
org.h2.jdbc.JdbcSQLException: Column "_ROWID_" not found; SQL statement:
CREATE FORCE VIEW PUBLIC._0 AS
SELECT
    _ROWID_
FROM TEST.PEOPLE [42122-195]
```
But, it OK in newer version.

Reply:
Can you look into the commit history and see the original purpose of adding this test?

Reply:
@cloud-fan https://github.com/apache/spark/commit/bfd3ee9f76aaab3dcde71d92e2b8ca60a0e42262#diff-f199a0c955304669dded23f71aabd41cd5e69680d91a007f5e3a37fb1d647c59

Reply:
OK seems `properties.setProperty("rowId", "false")` doesn't work for h2 2.0 anymore. I think we can remove this test now.

Reply:
OK